test/with_dummyserver/test_unix.py

```diff
@@ -0,0 +1,20 @@
+import os
+
+from dummyserver.testcase import HTTPDummyServerTestCase
+
+
+class UnixHTTPDummyServerTestCase(HTTPDummyServerTestCase):
+    host = '/tmp/dummyserver.sock'
+
+    @classmethod
+    def tearDownClass(cls):
+        cls._stop_server()
+        try:
+            os.remove(cls.host)
+        except OSError:
+            pass
+
+
+class TestUnixHTTPConnectionPool(UnixHTTPDummyServerTestCase):
```

Comment:
Can we somehow reuse the tests already written for our HTTPConnectionPool here? Would be great to have them generic enough to reuse.